### PR TITLE
[15.0] [MIG] purchase_reception_status

### PR DESCRIPTION
**Pull request by saxomanu**

### CODE REVIEW

**Review by vvrossem — APPROVED** [review]
Functional test : OK
Code review : LGTM (`oca-gen-addon-readme` should be reran as `README.rst` refers to `14.0`)

**Comment by vvrossem** [comment]
Hi @koenloodts , @alexis-via , @Rad0van , could you please review this PR ? 🙏 thank you

**Comment by vvrossem** [comment]
How to move this PR closer to be merged ? :)

**Review by Rad0van — CHANGES_REQUESTED** [review]
Functional test : OK
Code review : LGTM
As @vvrossem suggests: rerun oca-gen-addon-readme to update README.rst and index.html

**Comment by saxomanu** [comment]
@vvrossem @Rad0van README.rst and index.html are updated

File: purchase_reception_status/views/purchase_order.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            </field>
+        </field>
+    </record>
+    <record id="purchase_order_tree" model="ir.ui.view">
+        <field name="name">received_status.purchase.order.tree</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_tree" />
+        <field name="arch" type="xml">
+            <field name="invoice_status" position="before">
+                <field name="reception_status" />
```
Comment (CasVissers-360ERP) [inline purchase_reception_status/views/purchase_order.xml]: I think this should be aligned with invoice_status layout and new badge layout option:
       
```
 widget="badge"
 decoration-info="reception_status == 'partial'" decoration-success="reception_status == 'received'"/>
```

**Comment by CasVissers-360ERP** [comment]
@saxomanu Functional review, all good, minor layout feedback.

**Review by AungKoKoLin1997 — COMMENTED** [review]
Functional Review : Okay.
As @CasVissers-360ERP suggest: I also think it would be better UI if we use badge widget and align with invoice_status.

**Review by desdelinux — APPROVED** [review]
Functional Review is Ok

cc @HviorForgeFlow

File: purchase_reception_status/views/purchase_order.xml (hunk 1 above)
Comment (HviorForgeFlow) [reply purchase_reception_status/views/purchase_order.xml]: @saxomanu can you apply this? Thanks!

**Comment by HviorForgeFlow** [comment]
Please also squash readme upgrade commit

**Comment by desdelinux** [comment]
ping  @saxomanu

**Comment by saxomanu** [comment]
Add badge and squash commit

**Comment by desdelinux** [comment]
> Add badge and squash commit

@saxomanu pre-commit fail

**Comment by HviorForgeFlow** [comment]
@saxomanu pre-commit is failing.

Nitpick: Could you change [[FIX] add badge in tree views](https://github.com/OCA/purchase-workflow/pull/1355/commits/626f938f06c5bbd40b4b7b515791fe001be96bff) name for:
 
 [IMP] purchase_reception_status: Add badge in tree views
 
 Seems more an improvement, and module names help to future reads.
 
 Thanks!

**Pull request merged**